packages/flutter_tools/doc/attach.md

```diff
@@ -0,0 +1,33 @@
+# Flutter Attach
+
+## Overview
+
+A Flutter-command that attaches to applications that have been launched
+without `flutter run` and provides a HotRunner (enabling hot reload/restart).
+
+## Usage
+
+There are four ways for the attach command to discover a running app:
+
+1. If the app is already running and the observatory port is known, it can be
+explicitly provided to attach via the command-line, e.g. `$ flutter attach
+--debug-port 12345`
+1. If the app is already running and the platform is iOS, attach can use mDNS
+to lookup the observatory port via the application ID, with just `$ flutter
+attach`
+1. If the platform is Fuchsia and the app is not yet running, the module name
```

Comment:
This can also be invoked after the app is running

Reply:
updated